packages/hardhat-core/sample-projects/advanced/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  env: {
+    browser: false,
+    es2021: true,
+    mocha: true,
+    node: true,
+  },
+  extends: [
+    "standard",
+    "plugin:prettier/recommended",
+    "plugin:node/recommended",
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: "module",
```

Comment:
I'm not sure about this. We don't even know if hardhat supports ES6 modules.

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1632/commits/b60721da99dd4cb32c3f1e2371333915365b2820